CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Unreleased
-
+  
   - Prevent smart_open from writing to logs on import (PR [#476](https://github.com/RaRe-Technologies/smart_open/pull/476), [@mpenkov](https://github.com/mpenkov))
   - Modify setup.py to explicitly support only Py3.5 and above (PR [#471](https://github.com/RaRe-Technologies/smart_open/pull/471), [@Amertz08](https://github.com/Amertz08))
 
+# 1.10.1, 26 April 2020
```

Comment:
I guess we have a linearity problem. Do we arrange our versions by date (as you have done here) or by version? I think it's slightly confusing either way.

Reply:
By date – people look to latest changes on top.

Also, in case we ever make another critical bugfix release to 1.10.x, I'd expect to see the news on top, not hidden 5 pages down the list.